src/Spectre.Angular2Client/src/app/plotly/guid.service.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * guid.service.ts
+ * Random GUID generator.
+ *
+   Copyright 2017 Grzegorz Mrukwa
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class GuidService {
+
+  constructor() { }
+
+  next() {
```

Comment:
Can we provide some docstrings? Maybe *doxygen* will be able to generate something from such docstring, that will be useful for us in future?